detect_secrets/plugins/keyword.py

```diff
@@ -152,7 +152,8 @@
 OPTIONAL_WHITESPACE = r'\s*'
 OPTIONAL_NON_WHITESPACE = r'[^\s]{0,50}?'
 QUOTE = r'[\'"]'
-SECRET = r'[^\r\n]+'
+# Includes, at least, 1 alphanumeric character
+SECRET = r'[^\r\n]*[a-zA-Z0-9]+[^\r\n]*[^\r\n,\'"]'
```

Comment:
Do we want to classify backticks as quotes too?

Reply:
Yes, I like it. Done!